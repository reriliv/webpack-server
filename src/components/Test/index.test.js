import React from 'react';
// import renderer from 'react-test-renderer';
import { shallow } from 'enzyme';
import Test from './index';

describe('<Test />', () => {
  test('should render', () => {
    const wrapper = shallow(<Test>123</Test>);
    expect(wrapper.html()).toMatchSnapshot();
  });
});
