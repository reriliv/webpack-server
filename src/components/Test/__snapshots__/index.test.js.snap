// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Test /> should render 1`] = `"<div class=\\"test\\">123</div>"`;
